client/src/modules/reports/generate/balance_report/balance_report.html

```diff
@@ -0,0 +1,43 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 class="text-capitalize" translate>REPORT.BALANCE_REPORT.TITLE</h3>
+      <p class="text-info" translate>REPORT.BALANCE_REPORT.DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+
+        <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate autocomplete="off">
+
+          <!--select the period by a date-->
+          <bh-date-editor
```

Comment:
You should be able to simply add it.  The cash module uses.  [Here's the line](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/cash/cash.html#L64).